src/main/java/com/kodilla/ecommercee/service/UserDbService.java

```diff
@@ -19,4 +19,12 @@ public UserDbService(UserRepository userRepository) {
     public Optional<User> getUserById(Long id) {
         return userRepository.findById(id);
     }
+
+    public User getUser(Long userId) {
+        Optional<User> foundUser = userRepository.findById(userId);
+        if (foundUser.isPresent()) {
+            return foundUser.get();
+        }
+        return null;
```

Comment:
```suggestion
        return Optional.empty();
```
Ja bym użyła Optional.empty() - intelliJ podpowiada takie rozwiązanie i wygląda chyba lepiej.

Reply:
W tym miejscu moja metoda zawraca objekt User i chciałam zrezygonować z Optional. Ale jeśli tak jest profesjonalnie to zmienie. Andrzej poprosze o komentarz.

Reply:
Optional jest lepszy